### PR TITLE
Color cycle the body background

### PR DESCRIPTION
**Pull request by xyproto**
* Gradually change the background color, using CSS.
* Some refactoring: place JavaScript code in the `static/js`.

### CODE REVIEW

**Comment by ArthurG** [comment]
Downvote for giving you up :-1:

**Comment by xyproto** [comment]
Ok, not touching the redirect timing. Only color cycling and moving javascript code into files now.

**Comment by ArthurG** [comment]
30k timeout? :-1:

**Comment by xyproto** [comment]
@guoarthur No, the timeout is back to what it was: 3 seconds, which is the same as 3000 milliseconds. Check the previous JavaScript code and read the documentation for `setInterval`.

**Comment by lukeramsden** [comment]
Personally, I think the Rick Astley is really dumb, because at 3 seconds the entire page just becomes worthless.

**Comment by xyproto** [comment]
@lukeramsden See #157

File: server/index.html [hunk 1]
```diff
@@ -43,11 +43,6 @@
 
     <script src="//ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script> 
     <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
-    <script>
-        setTimeout(function(){
-            window.location = 'https://www.youtube.com/watch?v=dQw4w9WgXcQ';
-         }, 3000);
-    </script>
```
Comment (xyproto) [inline server/index.html:50]: This code is only moved to `rick.js`, not removed.

**Pull request closed without merging**